scraping/parser.py

```diff
@@ -29,8 +29,10 @@ class EventParser:
         # Indicates that a disk was low on space and disk space
         # could not be expanded.
         "FAILED_TO_EXPAND_DISK",
-        # Indicates that the cluster scoped init script has started.
+        # Indicates that the cluster scoped init script is starting.
         "INIT_SCRIPTS_STARTING",
+        # Indicates that the cluster scoped init script has started.
+        "INIT_SCRIPTS_STARTED",
```

Comment:
If Databricks introduces a new cluster event, all existing DAC installation will fail with the current implementation?

Reply:
It will fail. Currently we check if the event is valid because the parser doesn't have any direct connection to the Databricks API it just parse the incoming data structure. We can remove the check and trust the caller to send valid events.

Reply:
How about logging it with a warning and ignore the event? 
It's likely that Databricks will add more event types and we definitely don't want all the DAC installations to fail immediately.

Reply:
Good point, will implement.